packages/account/src/Components/trading-assessment/trading-assessment-dropdown.tsx

```diff
@@ -3,27 +3,55 @@ import classNames from 'classnames';
 import { Field } from 'formik';
 import { DesktopWrapper, Dropdown, MobileWrapper, Text, SelectNative } from '@deriv/components';
 import { localize } from '@deriv/translations';
+import { TFormData, TQuestion } from 'Types';
 import { MAX_QUESTION_TEXT_LENGTH } from '../../Constants/trading-assessment';
 
+type TradingAssessmentDropdownProps = {
+    disabled_items: string[];
+    item_list: TQuestion[];
+    onChange: (
+        e: React.ChangeEvent<HTMLSelectElement>,
+        form_control: keyof TFormData,
+        setFieldValue: (field: string, value: string, shouldValidate?: boolean) => void
+    ) => void;
+    values: TFormData;
```

Comment:
*Nit*: Can we use a type from deriv/api-types? If a type can be used, please replace all instances of TFormData